### ISSUE TITLE
Coveralls can't find branches for repository

### ISSUE THREAD

**Issue by soleares** [lemurheavy/coveralls-public#516]
1. Signed up for the trial.
2. Added a private repository.
3. Branches don't show up under Branch.
4. Click 'Sync Branches' and they still don't show up.
5. Additionally no coverage data is showing up in Coveralls from a successful CircleCI/slather run.

I also tried adding a public repository and I'm seeing the same behavior.

**Comment by nickmerwin** [comment]
Hi @soleares what repo is that? If you haven't already done-so, please setup a post build webhook for your circle builds:

https://coveralls.zendesk.com/hc/en-us/articles/203484329

**Comment by soleares** [comment]
@nickmerwin onemedical/mobile-ios. I'm sending the results with slather and circleci doesn't run parallel builds for iOS. Is the web hook still necessary?

**Comment by soleares** [comment]
@nickmerwin I submitted a ticket as I'd prefer to not share private repo info on a public forum.

**Comment by nickmerwin** [comment]
Ok thanks, will check there now.

**Comment by nickmerwin** [comment]
Hi @soleares I see a couple builds that have shown up in the last 20 minutes. Could you confirm it's good to go?

Right, if you're not building in parallel on Circle the webhook isn't necessary.

Thank you!

**Comment by soleares** [comment]
@nickmerwin It's working now. thanks!
